cpp/daal/src/algorithms/dtrees/forest/classification/df_classification_train_dense_default_impl.i

```diff
@@ -392,35 +409,42 @@ template <typename algorithmFPType, CpuType cpu>
 bool UnorderedRespHelper<algorithmFPType, cpu>::findBestSplitCategoricalFeature(const algorithmFPType * featureVal, const IndexType * aIdx, size_t n,
                                                                                 size_t nMinSplitPart, const algorithmFPType accuracy,
                                                                                 const ImpurityData & curImpurity, TSplitData & split,
-                                                                                double minWeightLeaf) const
+                                                                                const double minWeightLeaf, const algorithmFPType totalWeights) const
```

Comment:
double minWeightLeaf, algorithmFPType totalWeights

Reply:
fixed, double changed to algorithmFPType